db/migrate/20250121163154_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.2]
```

Comment:
When you're doing a take home project for an interview it's a great opportunity to think about database design:
1. Should any of these tables/fields have indexes?
2. Should any of these tables/fields have uniqueness constraints?
3. Is there a trade-off to using foreign key references directly/why did you use those instead of an ID field?

The schema doesn't have to be perfect but be ready to defend your choices.